magento/magento2
# Fixed #23982 Sales Configuration in the 'Orders Cron Settings' the field 'Pending Payment Order Lifetime (minutes)' have not proper validation

### Description (*)

Magento 2.3 Sales configuration in the `Sales` have `Orders Cron Settings` in the field  `Pending Payment Order Lifetime (minutes)` resolved proper validation.

### Fixed Issues (if relevant)

1. #23982 : Sales Configuration in the 'Orders Cron Settings' under the field 'Pending Payment Order Lifetime (minutes)' have not proper validation

### Manual testing scenarios (*)

1. Login to the Magento 2 admin.
2. Go to the STORES >> Settings >> Configuration >> Sales >> Orders Cron Settings >> Pending Payment Order Lifetime (minutes):
3. Pending Payment Order Lifetime (minutes): add any text value
4. Save Config
5. There should be allow only digits

### Questions or comments


### Contribution checklist (*)
